mobile-browser-based-version/src/helpers/model_definition/custom_layers.js

```diff
@@ -0,0 +1,86 @@
+import * as tf from '@tensorflow/tfjs';
```

Comment:
we could name the file sth like interop... to make clear it's specific to that?

Reply:
Yep, then maybe it's easier to have a folder called custom_layers in which we define interoperability_layer.js and where we can add more layers in case we need to define new custom layers.